### ISSUE TITLE
Cant install ?

### ISSUE THREAD

**Issue by unnamed666** [zendframework/zend-expressive-skeleton#152]
hi, 
i'am trying to start with Zend Expressive but after running 

`composer create-project zendframework/zend-expressive-skeleton expressive`

i get:

```
Problem 1
- Installation request for zendframework/zend-expressive-tooling ^0.4.1 -> satisfiable by zendframework/zend-expressive-tooling[0.4.1].
- zendframework/zend-expressive-tooling 0.4.1 requires zendframework/zend-component-installer ^0.7.1 -> satisfiable by zendframework/zend-component-installer[0.7.1] but these conflict with your requirements or minimum-stability.
```

and i'am stuck :)

**Comment by Ocramius** [comment]
@unnamed666 what settings did you pick while installing? I just tried locally and can't reproduce the problem.

Also, what PHP version are you running? The requirement has recently been bumped to PHP 7.1.x

**Comment by asgrim** [comment]
@unnamed666 try a `composer clear-cache` first, perhaps you have a stale package index from Composer? I also ran just fine (full log in gist: https://gist.github.com/asgrim/0cc8018cac4bd9313fb5aebe705610e6) with all default settings at least.

**Comment by unnamed666** [comment]
ok, so one more time i tried:
`composer clear-cache`
`composer create-project zendframework/zend-expressive-skeleton expressive`
all settings with Zend components and Whoops as error handler

and iam running
`PHP 7.0.6 (cli) (built: May 17 2016 12:47:25) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies`
on osx el capitan, php installed with brew

still getting same error as before

**Comment by xtreamwayz** [comment]
I'm getting the same issue. It's probably because of this: https://github.com/zendframework/zend-expressive-skeleton/blob/master/src/ExpressiveInstaller/config.php#L31

I'll create a PR for it.

**Comment by xtreamwayz** [comment]
Let me rephrase that. You get it when you install expressive with the modular structure.

The dependency conflict is caused by zend-expressive-tooling: https://github.com/zendframework/zend-expressive-tooling/blob/master/composer.json#L24

@unnamed666 Until a fixed is pushed, you can choose the flat structure. That one is working.

**Comment by xtreamwayz** [comment]
@unnamed666 This has been fixed now. Thanx for reporting.
